src/main/kotlin/org/elm/workspace/ElmWorkspaceService.kt

```diff
@@ -351,7 +355,12 @@ class ElmWorkspaceService(
         // Must load the Settings before the Elm Projects in order to have an ElmToolchain ready
         val settingsElement = state.getChild("settings")
         val binDirPath = settingsElement.getAttributeValue("binDirPath").takeIf { it.isNotBlank() }
-        modifySettings(notify = false) { RawSettings(binDirPath = binDirPath) }
+        val activateElmFormatOnSaveHook = settingsElement
+                .getAttributeValue("activateElmFormatOnSaveHook")
+                .takeIf { it != null && it.isNotBlank() }?.toBoolean()
+                ?: false
```

Comment:
another place where the default value is assumed to be falsed. This should be factored out with the others.